src/test/java/org/breedinginsight/api/v1/controller/TraitUploadControllerIntegrationTest.java

```diff
@@ -105,6 +115,9 @@ public void setup() {
 
         // Retrieve our new data
         validProgram = programDao.findAll().get(0);
+
```

Comment:
Any reason this was left in here commented out?

Reply:
removed and pushed